src/calculateRentalCost.js

```diff
@@ -4,7 +4,28 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  let carRent = PRICE_PER_DAY * days;
+
+  if (days >= LONG_TERM) {
+    carRent = carRent - LONG_TERM_DISCOUNT;
+
+    return carRent;
+  }
+
+  if (days >= SHORT_TERM) {
+    carRent = carRent - SHORT_TERM_DISCOUNT;
+
+    return carRent;
```

Comment:
The logic for applying discounts is incorrect. If the number of days is greater than or equal to `LONG_TERM`, the function returns immediately after applying the long-term discount, without checking for the short-term discount. This means that the short-term discount will never be applied if the long-term condition is met. You should adjust the logic to ensure that the correct discount is applied based on the number of days.